### DISCUSSION TITLE
Unable to create circuit terminations with interface via API

### DISCUSSION THREAD

**Discussion by Shailendrasnegi** [netbox-community/netbox#9799] [Q&A]
Hi,
Please support as i am unable to create circuit terminations with interface via API. 
i have created ckt ID and Termination A and Z via API but bow wants to add interface at either Termination point.
How can we achieve this?

Netbox Version: v3.2.5
python: 3.8.10

![Screenshot from 2022-07-20 17-26-52](https://user-images.githubusercontent.com/78404159/179976138-9ae048c4-4ba3-47ba-b813-807d90e15aaf.png)

**Comment by candlerb** [comment]
You should be able to create a _cable_ between an interface and a circuit termination.

**Reply by Shailendrasnegi** [reply]
Can we do it via API ? I want to achieve this (as per attached image) via API,s instead of doing manually on netbox.
I have to connect Interface via API's with circuit endpoints A or Z.
![Screenshot from 2022-07-21 10-00-44](https://user-images.githubusercontent.com/78404159/180130235-6e1ebd4a-1e26-4696-88ad-07a13e356f8b.png)

**Reply by candlerb** [reply]
> Can we do it via API ?

I believe you should be able to. What code have you tried, and what error did you get back?

**Reply by Shailendrasnegi** [reply]
Hi @candlerb ,
Thanks for support. Issue has resolved as per solution you have provided. After circuit endpoint, I used 'post api' for cable creation.
Thanks once again.
